src/core/data.jl

```diff
@@ -386,6 +386,7 @@ function _rescale_cost_model(comp::Dict{String,Any}, scale::Real, multiphase::Bo
         else
             phases = length(comp["model"])
             for ph in 1:phases
+                ph_str = phases > 1 ? " on phase $(ph)" : ""
```

Comment:
Seems reasonable.  Consider putting the string generation into a function to try to enforce consistency.

Note that _implicit single phase_ occurs when the "phases" term is omitted from the data.  If the "phases" term is set to 1, then it is _explicit single phase_ and it is still ok to print "on phase 1".

Reply:
I did think about a function, but the wording of the warnings were not conducive to always using the same string, therefore making a function wouldn't save much duplication.

In that case, I will use `haskey(data, "phases")` in most cases. In the case you quoted above, `data` isn't passed to the function, so I will just check the type of `comp["model"]`